content/general-docs/introduction.adoc

```diff
@@ -3,11 +3,81 @@
+As a result, previous simulations were primarily used during the early stages of development, where precise physical modeling was not necessarily required. 
+
+With increased computing power and streamlined simulation techniques, virtual testing plays a more powerful role at all stages of system development.
+This correlates with the growing need to accurately represent the real world in simulations, so-called digital twins, especially as the need for physical sensor simulation increases. 
+
+One fundamental step to describing the real world virtually is a full physical description of all objects, including their material properties and their 3D geometries.
+
+It is extremely important to standardize the representation of data and attributes, such as scenarios, road data, and asset data, necessary for AD/ADAS simulation in order to promote the circulation of these data and to obtain consistent results among different systems. 
+
+Standardization activities have been conducted for scenarios and road data using OpenSCENARIO, OpenDRIVE, OpenCRG, and other standards.
+ASAM OpenMATERIAL closes the gap to advance standardization by providing standardizes definitions of material properties and 3D geometry.
+
+=== The two parts of OpenMATERIAL
+
+ASAM OpenMATERIAL has two main parts.
+
+* The Material part contains definitions and file formats to store and exchange material properties. These can either be physical properties, such as surface roughness, permittivity, index of refraction, etc., or measured data, such as wavelength and angle depended reflectance values.
+* The Geometry part contains node hierarchies of different objects classes.
+These structures contain a uniform naming scheme and coordinate definitions to enable the exchange of 3D models between different simulation systems.
+
+=== How everything fits together
+
+ASAM OpenMATERIAL specifies different file formats for the definition of assets, material properties and the mapping between the two.
+<<fig-openmaterial-overview>> visualizes the different file formats and their connections.
+
+[#fig-openmaterial-overview]
+.Overview of ASAM OpenMATERIAL file formats and data flow
+image::openmaterial_overview.png[,1100]
+
+The first touching point with an OpenMATERIAL compliant 3D model is the asset file.
+
+The *asset file* is a JSON file with the file extension .xoma (OpenMATERIAL Asset).
+A detailed definition of the https://github.com/asam-ev/OpenMATERIAL/blob/main/schemas/asset_schema.json[asset file JSON schema] is given in xref:geometry/asset-schema.adoc[].
+It contains metadata, such as description, unique identifier, version information, copyright, etc.
+The asset files also contains a material texture assignment table.
+In this table, dedicated material mapping textures can be assigned to materials within a 3D data files.
+This provides to possibility to map comprehensive material properties to the geometry on a texture level.
```

Comment:
Typo: "This provides "the" possibility...

Reply:
Thanks, I fixed it.